.gitpod.yml

```diff
@@ -0,0 +1,31 @@
+tasks:
+  - name: Stargate
+    command: docker-compose up stargate
+  - name: api
+    command: |
+      sleep 35
+      docker-compose up eddiehub-api
+  - name: Eddiebot
+    init: |
+      npm i -g npm
+      npm install
+      gp await-port 3000
+    command: npm run start:dev
+
+github:
+  prebuilds:
+    # enable for the default branch (defaults to true)
+    master: true
+    # enable for all branches in this repo (defaults to false)
+    branches: true
+    # enable for pull requests coming from this repo (defaults to true)
+    pullRequests: true
+    # enable for pull requests coming from forks (defaults to false)
+    pullRequestsFromForks: true
+    # add a check to pull requests (defaults to true)
+    addCheck: true
+    # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
+    addComment: true
+    # add a "Review in Gitpod" button to the pull request's description (defaults to false)
+    addBadge: false
```

Comment:
This will add a button like this.

<img src="https://gitpod.io/button/open-in-gitpod.svg"/>


```suggestion
    addBadge: true
```